pkg/autodiscovery/autoconfig.go

```diff
@@ -256,7 +256,7 @@ func (ac *AutoConfig) GetAllConfigs() []integration.Config {
 	for _, pd := range ac.providers {
 		cfgs, err := pd.provider.Collect()
 		if err != nil {
-			log.Debugf("Unexpected error returned when collecting configurations from provider %v: %v", pd.provider, err)
+			log.Errorf("Unexpected error returned when collecting configurations from provider %v: %v", pd.provider, err)
```

Comment:
@DataDog/container-integrations This is to match the severity of 

https://github.com/DataDog/datadog-agent/blob/6.6/pkg/autodiscovery/autoconfig.go#L608

Is that ok for you ?